recipes/baumwelch/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "baumwelch" %}
+{% set version = "0.3.0" %}
+
+package: 
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.openfst.org/twiki/pub/Contrib/FstContrib/{{ name }}-{{ version }}.tar.gz
+  sha256: 01c2cc16c5eedcb28cdc2ae83659fc95bab0a0d32a7e9209fbecd59b8747d0f7
+  
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - openfst ==1.7.5
+  run:
+    - openfst ==1.7.5
+
+test:
+  commands:
+    - baumwelchtrain --helpshort >/dev/null || [[ $? == 1 ]]
```

Comment:
You don't need to redirect to /dev/null, and generally recommend against it.  Also, does this really provide a non-zero exit code on success?

Reply:
I removed the redirect.

Yes, the --help and --helpshort methods return 1; other errors return (including not found) something else. It came as a surprise to me too writing these.

Reply:
That is insane!  That should be considered a bug upstream